crypto/CMakeLists.txt

```diff
@@ -7,20 +7,38 @@ project(StarkwareCryptoLib VERSION 0.1.0 LANGUAGES CXX)
 
 set(CMAKE_CXX_STANDARD 17)
 
+set(BOOST_ENABLE_CMAKE ON)
+
 include(FetchContent)
 FetchContent_Declare(
   googletest
   URL https://github.com/google/googletest/archive/609281088cfefc76f9d0ce82e1ff6c30cc3591e5.zip
 )
+FetchContent_Declare(
+  boostorg
+  URL https://boostorg.jfrog.io/artifactory/main/release/1.79.0/source/boost_1_79_0.tar.gz
+)
 # For Windows: Prevent overriding the parent project's compiler/linker settings
 set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
 FetchContent_MakeAvailable(googletest)
 
+FetchContent_GetProperties(boostorg)
+if (NOT boostorg_POPULATED)
+    FetchContent_Populate(boostorg)
+endif ()
+
+include_directories(${boostorg_SOURCE_DIR})
```

Comment:
Is it possible to fetch it conditionally based on compilation target?

Reply:
Yes, I've implemented it in newest commit.